pkg/controller/clusterpool/clusterpool_controller_test.go

```diff
@@ -468,6 +470,11 @@ func TestReconcileClusterPool(t *testing.T) {
 
 			for _, cd := range cds.Items {
 				assert.Equal(t, hivev1.HibernatingClusterPowerState, cd.Spec.PowerState, "expected cluster to be hibernating")
+				if test.expectedLabels != nil {
+					for k, v := range test.expectedLabels {
+						assert.Equal(t, v, cd.Labels[k])
+					}
+				}
```

Comment:
```
				assert.Equal(t, test.expectedLabels, cd.Labels, "unexpected labels on cluster deployment")
```

Reply:
This does pass today, but I didn't want to encode the assumption that these would be the exact and only labels on the resulting clusterdeployment. We inject labels for region and version, that isn't happening in this test right now but it seems more correct to assert just the labels we expect are there, rather than assume they'll be the only ones.

Reply:
:+1: